src/parser/core/modules/CooldownDowntime.js

```diff
@@ -14,170 +14,131 @@ export default class CooldownDowntime extends Module {
 		'downtime',
 	]
 
-	//Downtime Actions IDs
-	_downTime = {}
+//Downtime Actions IDs
```

Comment:
Something has gone super janky here - can't really compare while it's like this.